ASP.NET API/WAVC_WebApi/Controllers/Identity/LoginController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+using WAVC_WebApi.Models;
+using WAVC_WebApi.Models.AuthenticationModels;
+
+namespace WAVC_WebApi.Controllers.Identity
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class LoginController : ControllerBase
+    {
+        private readonly SignInManager<ApplicationUser> _signInManager;
+
+        public LoginController(SignInManager<ApplicationUser> signInManager)
+        {
+            _signInManager = signInManager;
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> SignIn(LoginModel input)
+        {
+            var result = await _signInManager.PasswordSignInAsync(input.Email, input.Password, input.RememberMe, lockoutOnFailure: true);
+            if (!result.Succeeded)
+            {
+                return BadRequest();
```

Comment:
If signing in failed you should retrieve error message from `_signInManager` and pass it to BadRequest as a string argument.

Reply:
The only error we support right now is in case that credentials were wrong, so it's obvious. Thanks to that I could keep your changes on client.